js-api-doc/index.d.ts

```diff
@@ -15,6 +15,8 @@ export {
   FileImporter,
   Importer,
   ImporterResult,
+  nodePackageImporter,
+  NodePackageImporter,
```

Comment:
@nex3 Are there examples of other classes we want to document but not export? If I don't export NodePackageImporter here, `typedoc` can't include it in the docs, and throws a warning- `[warning] NodePackageImporter, defined in ./js-api-doc/importer.d.ts, is referenced by nodePackageImporter but not included in the documentation.`

One option would be to add it to `[intentionallyNotExported](https://typedoc.org/options/validation/#intentionallynotexported)`, which would silence the warning. We would need to document what the `NodePackageImporter` type is within the `nodePackageImporter` and `Options.importers` documentation.

Reply:
Let's just export the type. It makes sense that users might want to reference it in a union type or something at some point.